test/parallel/test-v8-serdes.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
```

Comment:
A test that includes a pre-serialized buffer that can be round-tripped serialized->deser->re-serialized would be good also.